pyspark/test/bigdl/keras/test_layer.py

```diff
@@ -309,6 +309,8 @@ def test_reshape(self):
         input_data = np.random.random_sample([1, 3, 5, 4])
         layer = Reshape(target_shape=(3, 20), input_shape=(3, 5, 4))
         self.modelTestSingleLayer(input_data, layer)
+        layer2 = Reshape(target_shape=(-1, 2, 3), input_shape=(3, 5, 4))
```

Comment:
what if there's multiple -1 ?

Reply:
This is a Keras layer, and what we do on python side is under the assumption that we write correct code in Keras. If there are multiple -1, Keras will handle the expection.